DEVELOPING.md

```diff
@@ -24,10 +24,10 @@ First upgrade pip and then install the dependencies
 Get a client ID and secret, if you don't already have one.
 
 ```
-(.venv) $ curl --request POST --header "Content-Type: application/json" --data '{"redirect_uris": ["http://localhost:1234", "http://localhost:8000/oidc/callback/"], "application_type": "native", "token_endpoint_auth_method": "client_secret_post"}' https://iddev.fedorainfracloud.org/openidc/Registration
```

Comment:
Hm, I kinda like the raw curl example too. Is there any harm to leaving both examples in?

Reply:
No :), I ll leave it in